api/orchestrator/orchestrator.proto

```diff
@@ -657,17 +657,18 @@ message CloudService {
 
   message Metadata {
     // a map of key/value pairs, e.g., env:prod
-   map<string, string> labels = 1;
+   map<string, string> labels = 1 ;
    
    // an icon for the cloud service used by the UI
    string icon = 2;
```

Comment:
I would probably make all of them optional and avoid defining a "default color" in the backend. This is something we can set as a default in the frontend.